.github/scripts/generate_ci_workflows.py

```diff
@@ -503,8 +503,8 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
     ),
     CIWorkflow(
         arch="linux",
-        build_environment="linux-xenial-cuda11.3-py3.7-gcc7",
-        docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-xenial-cuda11.3-cudnn8-py3-gcc7",
+        build_environment="linux-xenial-cuda11.3-py3.10-gcc7",
```

Comment:
Why are we changing py3.7 workflow instead of adding new 3.10 workflow ?

Reply:
Because there are other python-3.7 cuda enabled configs, aren't they?

Reply:
Yes true we do have multiple 3.7 configs